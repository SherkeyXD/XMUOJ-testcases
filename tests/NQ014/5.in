4
26
8
10
20
3
5
12
