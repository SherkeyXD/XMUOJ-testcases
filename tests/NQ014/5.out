22
100663294
382
1534
1572862
10
46
6142
